### PR TITLE
add sompy and portalocker

### PR DESCRIPTION
**Pull request by daler**
These are deps for https://github.com/BaselAbujamous/clust which will then be added to bioconda.

### CODE REVIEW

File: recipes/portalocker/bld.bat [hunk 1]
```diff
@@ -0,0 +1,2 @@
+"%PYTHON%" setup.py install
```
Comment (synapticarbors) [inline recipes/portalocker/bld.bat]: Can we get rid of the `bld.bat` and `build.sh` in favor of the `script` entry in the `meta.yaml` files:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L29

File: recipes/portalocker/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+{% set sha256 = "b955d6197c7cd3206500d3237a2dd6cc899ceebb13460cc97e658cbab70df11b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+sha256: {{ sha256 }}
```
Comment (synapticarbors) [inline recipes/portalocker/meta.yaml]: Indentation of `sha256`

File: recipes/portalocker/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+requirements:
+  build:
+    - python
```
Comment (synapticarbors) [inline recipes/portalocker/meta.yaml]: Can you add `setuptools` to the `build` requirements?

File: recipes/portalocker/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+test:
+  imports:
+    - portalocker
+
+about:
+  home: http://portalocker.readthedocs.org/en/latest/
+  license: PSF
+  summary: 'Portalocker is a library to provide an easy API to file locking.'
```
Comment (synapticarbors) [inline recipes/portalocker/meta.yaml]: `dev_url: https://github.com/WoLpH/portalocker`

File: recipes/portalocker/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+  run:
+    - python
+
+test:
+  imports:
+    - portalocker
+
+about:
+  home: http://portalocker.readthedocs.org/en/latest/
+  license: PSF
```
Comment (synapticarbors) [inline recipes/portalocker/meta.yaml]: `license_file: LICENSE`

File: recipes/portalocker/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+  summary: 'Portalocker is a library to provide an easy API to file locking.'
+
+extra:
+  recipe-maintainers:
```
Comment (synapticarbors) [inline recipes/portalocker/meta.yaml]: @johanneskoester and @bgruening -- are you ok being listed as co-maintainers of the conda-forge recipes for `portalocker` and `sompy`?

File: recipes/sompy/bld.bat [hunk 7]
```diff
@@ -0,0 +1,2 @@
+"%PYTHON%" setup.py install --single-version-externally-managed --record record.txt
```
Comment (synapticarbors) [inline recipes/sompy/bld.bat]: Drop `bld.bat` and `build.sh` in favor of `script` entry in `meta.yaml`

File: recipes/sompy/meta.yaml [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+{% set sha256 = "846626a39e27cca171a81f4388cb30a32cdcd3bedf170c91227cf5a958aaabec" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+sha256: {{ sha256 }}
```
Comment (synapticarbors) [inline recipes/sompy/meta.yaml]: Indentation of `sha256`

File: recipes/sompy/meta.yaml [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy
```
Comment (synapticarbors) [inline recipes/sompy/meta.yaml]: It doesn't look like `numpy` is a `build` requirement if using `--single-version-externally-managed`

File: recipes/sompy/meta.yaml [hunk 10]
```diff
@@ -0,0 +1,38 @@
...
+
+  run:
+    - python
+    - numpy
```
Comment (synapticarbors) [inline recipes/sompy/meta.yaml]: It looks like `matplotlib` is also a `run` requirement. It's not evident when looking at the code up on github, but in the tarball version of the code, it's imported:

https://github.com/ttlg/sompy/commit/1efcb984132424784282f929362421001df1c4a3#diff-8320b10d61a88875c3c04a410d7405caL3

File: recipes/sompy/meta.yaml [hunk 11]
```diff
@@ -0,0 +1,38 @@
...
+
+test:
+  imports:
+    - sompy
+
+about:
+  home: https://github.com/ttlg/sompy
+  license: MIT License
```
Comment (synapticarbors) [inline recipes/sompy/meta.yaml]: `license: MIT`
  
Also, please manually package the `license_file` and submit an issue/PR upstream, adding the url to the issue as a comment in the recipe. 

https://conda-forge.org/docs/meta.html#packaging-the-license-manually

**Comment by sodre** [comment]
FYI: portalocker was added to conda-forge in #5092.

**Comment by apcamargo** [comment]
I added Clust to Bioconda in this PR: https://github.com/bioconda/bioconda-recipes/pull/11619

@npavlovikj thoughtfully submitted a Sompy recipe to Bioconda so that my PR could be merged, but I agree that it fits conda-forge better.

**Pull request closed without merging**